src/article.py

```diff
@@ -31,4 +33,15 @@ def serialize(self):
             "publicationSlug": self.publication["slug"],
             "publication": self.publication,
             "title": self.entry.title,
+            "filtered": self.isProfane(),
         }
+
+    def isProfane(self):
+        content = ""
+        check_content = False
+        try:  # Sometimes the entry doesn't have a content field
+            content = self.entry.content[0]["value"]
+            check_content = pf.contains_profanity(content)
+        except:
+            pass
+        return pf.contains_profanity(self.entry.title) or check_content
```

Comment:
do not use try excepts -- this isn't an ideal situation to be in -- you should instead use `.get` or catch more specific exceptions